companion-api/backend/etcd/etcd.go

```diff
@@ -64,27 +64,29 @@ func (e *etcdClient) CreateUser(id string, user *backend.UserData) (err error) {
 	return
 }
 
-func (e *etcdClient) UpdateUser(id string, userToUpdate *backend.UserData, update func(old, new *backend.UserData)) (err error) {
+func (e *etcdClient) UpdateUser(id string, patchs []backend.UserDataPatch) (err error) {
```

Comment:
I still want my signature, even if it was patching only (which it won't be), the update method would actually apply the patch, not this one.